src/layman/layer/filesystem/gdal.py

```diff
@@ -200,6 +200,19 @@ def get_statistics(filepath):
     return result
 
 
+def get_file_list_statistics(filepaths):
+    stats_list = [get_statistics(filepath) for filepath in filepaths]
+    bands_count = len(stats_list[0])
+    stats = []
+    for i in range(0, bands_count):
+        ongoing_band_min, ongoing_band_max = stats_list[0][i][0], stats_list[0][i][1]
+        for file_stats in stats_list:
+            ongoing_band_min = min(ongoing_band_min, file_stats[i][0])
+            ongoing_band_max = max(ongoing_band_max, file_stats[i][1])
+        stats.append([ongoing_band_min, ongoing_band_max])
```

Comment:
Either stop returning mean and deviation from get_statistics, or return NamedTuple from both get_statistics and get_file_list_statistics.